File: Dolphin/ControlGroup/Code/Dev/ProdSparkService/src/main/java/com/mnrclara/spark/core/model/PickupHeader.java
package com.mnrclara.spark.core.model;

import jdk.jfr.DataAmount;
import lombok.Data;

import java.sql.Timestamp;

@Data
public class PickupHeader {
    private String languageId;
    private String companyCodeId;
    private String plantId;
    private String warehouseId;
    private String preOutboundNo;
    private String refDocNumber;
    private String partnerCode;
    private String pickupNumber;
    private Long lineNumber;
    private String itemCode;
    private String proposedStorageBin;
    private String proposedPackBarCode;
    private Long outboundOrderTypeId;
    private Double pickToQty;
    private String pickUom;
    private Long stockTypeId;
    private Long specialStockIndicatorId;
    private String manufacturerPartNo;
    private Long statusId;
    private String assignedPickerId;
    private String referenceField1;
    private String referenceField2;
    private String referenceField3;
    private String referenceField4;
    private String referenceField5;
    private String referenceField6;
    private String referenceField7;
    private String referenceField8;
    private String referenceField9;
    private String referenceField10;
    private Long deletionIndicator;
    private String remarks;
    private String pickupCreatedBy;
    private Timestamp pickupCreatedOn;
    private String pickConfimedBy;
    private Timestamp pickConfimedOn;
    private String pickUpdatedBy;
    private Timestamp pickUpdatedOn;
    private String pickupReversedBy;
    private Timestamp pickupReversedOn;
}

File: MobileApp/Almailem/app/src/main/java/com/tvhht/myapplication/picking/model/PickUpHeader.kt
package com.tvhht.myapplication.picking.model

import com.google.gson.annotations.SerializedName


data class PickUpHeader(
    @SerializedName("warehouseId") val warehouseId: List<String>? = null,
    @SerializedName("companyCodeId") val companyCodeIdList: List<String>? = null,
    @SerializedName("plantId") val plantIdList: List<String>? = null,
    @SerializedName("languageId") val languageIdList: List<String>? = null,
    @SerializedName("levelId") val levelIdList: List<Int>? = null,
    @SerializedName("statusId") val statusId: List<Int>? = null,
    @SerializedName("assignedPickerId") val assignedPickerIdList: List<String>? = null,
    @SerializedName("OutboundOrderTypeId") val outboundOrderTypeId: List<Int>? = null
)
